modules/addBook.js

```diff
@@ -0,0 +1,21 @@
+import displayBooks from './display.js';
+import {Books} from '/index.js'; 
+import {booksList} from '/index.js'; 
+
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+const errorMesg = document.querySelector('.error-mesg');
+
+  export default function addBook() {
```

Comment:
- [ ] Great function implementation, but as the project required, kindly change that to an arrow function so it can follow the ES6 best practices.

Reply:
This change has been implemented well